### PR TITLE
chore: upgrade nx to latest

### PR DESCRIPTION
**Pull request by Itrulia**
ToDo:

- [x] Extensive testing on the editor due to graphql upgrade
- [x] Testing if without cors setting on V2 API we can still make calls to it with the authorization header

Had to re-add cors, now it's set globally like it used to be with REST.

### CODE REVIEW

**Comment by Itrulia** [comment]
Thank you very much @mandarini for the help debugging and cleanup of the storybook settings!

File: apps/api-example/src/nestapp/app.module.ts [hunk 1]
```diff
@@ -27,11 +27,8 @@ import {JobsModule} from '@wepublish/jobs'
       autoSchemaFile: './apps/api-example/schema-v2.graphql',
       sortSchema: true,
       path: 'v2',
-      cors: {
```
Comment (Itrulia) [inline apps/api-example/src/nestapp/app.module.ts:30]: Seem to have been removed

File: apps/api-example/webpack.config.js
Comment (Itrulia) [inline apps/api-example/webpack.config.js:1]: Enables SWC compilation of the Nestjs application

File: libs/api/src/lib/graphql/action.ts [hunk 2]
```diff
@@ -11,15 +11,15 @@ import {GraphQLUser} from './user'
 import {GraphQLEvent} from './event/event'
 
 const actionFields = {
-  date: {type: GraphQLNonNull(GraphQLDateTime)}
+  date: {type: new GraphQLNonNull(GraphQLDateTime)}
```
Comment (Itrulia) [inline libs/api/src/lib/graphql/action.ts:14]: Now only works via constructors

File: libs/api/src/lib/graphql/setting.ts [hunk 3]
```diff
@@ -46,7 +46,15 @@ export const GraphQLSettingName = new GraphQLEnumType({
 export const GraphQLAllowedSettingVals = new GraphQLObjectType({
   name: 'AllowedSettingVals',
   fields: {
-    stringChoice: {type: GraphQLList(GraphQLString)},
+    stringChoice: {type: new GraphQLList(GraphQLString)},
+    boolChoice: {type: GraphQLBoolean}
+  }
+})
+
+export const GraphQLInputAllowedSettingVals = new GraphQLInputObjectType({
```
Comment (Itrulia) [inline libs/api/src/lib/graphql/setting.ts:54]: Needs an InputObjectType now for inputs. Not a breaking change as only for admin API

**Review by antkiewiczk — APPROVED** [review]

File: apps/website-example/next.config.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ const nextConfig = {
   compiler: {
     // This is needed so that we can use components as selectors in Emotion
     emotion: {
+      sourceMap: true,
```
Comment (Itrulia) [inline apps/website-example/next.config.js:21]: See https://github.com/mui/material-ui/issues/38359

**Pull request merged**